src/rgw/rgw_user.cc

```diff
@@ -542,6 +542,17 @@ uint32_t rgw_str_to_perm(const char *str)
   return RGW_PERM_INVALID;
 }
 
+int rgw_is_valid_tenant(const string& t){
```

Comment:
i think `rgw_validate_tenant_name` is a better fit for this interface, since it either returns 0 or `-ERR_INVALID_TENANT_NAME`. seeing the name `is_valid_tenant`, i would expect it to return true/false